src/Containers/MiniPalette/MiniPalette.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { expandMiniPalette } from '../../Actions';
+
+export class MiniPalette extends Component{
+  expandPalette = () => {
+    const { color1, color2, color3, color4, color5 } = this.props
+    const colors = [color1, color2, color3, color4, color5]
+    this.props.expandMiniPalette(colors)
+  }
+
+  render() {
```

Comment:
Great work! Having the miniPalettes display with their colors really adds to the user experience